openquake/risklib/workflows.py

```diff
@@ -566,6 +567,9 @@ def __call__(self, loss_type, assets, ground_motion_values, epsilons,
         average_losses = loss_matrix.sum(axis=1) * self.ses_ratio
         values = get_values(loss_type, assets)
         ela = loss_matrix.T * values  # matrix with T x N elements
+        cb = self.riskmodel.curve_builders[self.riskmodel.lti[loss_type]]
+        # ugly workaround for qa_tests.event_based_test in Ubuntu 12.04
```

Comment:
Please add a `FIXME` co we can easly find it when support for 12.04 will bi removed